examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,86 @@
+"""
+============================
+Classifier Chain
+============================
+
+Demonstrate that a single classifier chain
+out performs 10 independent classifiers
+and that an ensemble of 10 randomly ordered
+classifier chains out performs a single classifier chain
+
+"""
+
+import numpy as np
+from sklearn.multi_label import ClassifierChain
+from sklearn.model_selection import train_test_split
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.metrics import jaccard_similarity_score
+from sklearn.linear_model import LogisticRegression
+from sklearn.datasets import fetch_rcv1
+
+# Get the Reuters Corpus Volume I dataset
+rcv1 = fetch_rcv1()
```

Comment:
Yeah. I was worried about that. With my connection I believe it took a couple minutes to download. Of course that time is depends on connection speed. But it doesn't seem like something that should be run automatically.

Previously I was using **make_multilabel_classification** to generate a fake dataset which is obviously much faster. But I don't think this works because I believe the labels in the fake dataset are uncorrelated which means there is no advantage to using a classifier chain.